indicators/banco-de-crédito-del-perú-xejfkd.yml

```diff
@@ -0,0 +1,38 @@
+title: Banco de Crédito del Perú (BCP) Phishing Kit XEjFkd
+description: |
+    Detects a phishing kit targeting Banco de Crédito del Perú (BCP). BCP is the largest bank in Peru.
+    This was found as a result of this kit being deployed on Replit.
+
+
+references:
+    - https://urlscan.io/result/bf07a311-9c17-4538-8396-2b2cd7b1a917/
+    - https://urlscan.io/result/a17da382-1ace-4012-9a81-06eab1c365d2/
+    - https://urlscan.io/result/e993630d-6656-47ee-871e-c970ebda7e09/
+
+detection:
+
+    title:
+      html|contains:
+        - <title>Banco de Crédito BCP</title>
```

Comment:
A better indicator may be the unique `origin-trial` token as it is specific to a domain,
the latter segment of the origin trial token contains metadata about the trial being used:

```mermaid
graph TD;
A751Xsk4ZW3DVQ8WZng2Dk5s3YzAyqncTzgv+VaE6wavgTY0QHkDvUTET1o7HanhuJO8lgv1Vvc88Ij78W1FIAAAAAB7eyJvcmlnaW4iOiJodHRwczovL3d3dy5nb29nbGV0YWdtYW5hZ2VyLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjgwNjUyNzk5LCJpc1RoaXJkUGFydHkiOnRydWV9-->A751Xsk4ZW3DVQ8WZng2Dk5s3YzAyqncTzgv+VaE6wavgTY0QHkDvUTET1o7HanhuJO8lgv1Vvc88Ij78W1FIAAAAAB7;
        A751Xsk4ZW3DVQ8WZng2Dk5s3YzAyqncTzgv+VaE6wavgTY0QHkDvUTET1o7HanhuJO8lgv1Vvc88Ij78W1FIAAAAAB7eyJvcmlnaW4iOiJodHRwczovL3d3dy5nb29nbGV0YWdtYW5hZ2VyLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjgwNjUyNzk5LCJpc1RoaXJkUGFydHkiOnRydWV9-->eyJvcmlnaW4iOiJodHRwczovL3d3dy5nb29nbGV0YWdtYW5hZ2VyLmNvbTo0NDMiLCJmZWF0dXJlIjoiUHJpdmFjeVNhbmRib3hBZHNBUElzIiwiZXhwaXJ5IjoxNjgwNjUyNzk5LCJpc1RoaXJkUGFydHkiOnRydWV9;

```

```json
{
  "origin": "https://www.googletagmanager.com:443",
  "feature": "PrivacySandboxAdsAPIs",
  "expiry": 1680652799,
  "isThirdParty": true
} 
```

There also appears to be a unique file that is pulled in as JavaScript code from `./data/f.txt` so this may also be a better indicator in terms of uniqueness.